eng/testing/linker/project.csproj.template

```diff
@@ -15,6 +15,8 @@
     <TargetingPackDir>{TargetingPackDir}</TargetingPackDir>
     <NETCoreAppMaximumVersion>{NetCoreAppMaximumVersion}</NETCoreAppMaximumVersion>
     <MicrosoftNETCoreAppVersion>{MicrosoftNETCoreAppVersion}</MicrosoftNETCoreAppVersion>
+    <!-- Workaround issue: https://github.com/dotnet/runtime/issues/55600 -->
+    <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
```

Comment:
This can be reverted now that https://github.com/dotnet/runtime/pull/55756 is merged.